src/uct/ib/base/ib_iface.h

```diff
@@ -526,14 +526,15 @@ size_t uct_ib_iface_get_max_iov(uct_ib_iface_t *iface)
 
 
 static UCS_F_ALWAYS_INLINE
-void uct_ib_iface_set_max_iov(uct_ib_iface_t *iface, size_t max_iov)
+void uct_ib_iface_set_max_iov(uct_ib_iface_t *iface, size_t max_iov,
+                              size_t trim_thresh)
 {
     size_t min_iov_requested;
 
     ucs_assert((ssize_t)max_iov > 0);
 
-    min_iov_requested = ucs_max(max_iov, 1UL); /* max_iov mustn't be 0 */
-    iface->config.max_iov = ucs_min(UCT_IB_MAX_IOV, min_iov_requested);
+    min_iov_requested     = ucs_max(max_iov, 1UL); /* max_iov mustn't be 0 */
+    iface->config.max_iov = ucs_min(trim_thresh, min_iov_requested);
```

Comment:
how does it work that we have single max_iov field in iface but different max_iov for put/get/am (at least for mlx5 case)?

Reply:
in mlx5 `uct_ib_iface_get_max_iov` is use for put/get only

Reply:
so pls rename accordingly or move this field down to verbs/mlx5 to avoid confusion (btw i don't think we need such field in mlx5 at all since we can calculate it during query)